frontend/plugins/accounting_ui/src/modules/transactions/transaction-form/components/helpers/CurrencyForm.tsx

```
@@ -79,18 +79,20 @@ export const CurrencyForm = ({
   };
 
   useEffect(() => {
-    if (
-      spotRate &&
-      amount &&
-      amount !== spotRate * (detail.currencyAmount || 0)
-    ) {
+    form.setValue(`trDocs.${journalIndex}.details.0.amount`, spotRate * (detail.currencyAmount ?? 0));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)